app/controllers/application_controller.rb

```diff
@@ -7,4 +7,8 @@ def set_current_user
     Current.user = User.find_by(id: session[:user_id]) if session[:user_id]
   end
 
+  def is_admin?
```

Comment:
Do you need this method in controller as well? If not you can define this as a helper method and then directly call the method in views.